keymirror/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for keymirror 0.1.1
+// Project: https://github.com/STRML/keyMirror
+// Definitions by: Johannes Fahrenkrug <https://github.com/jfahrenkrug>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function KeyMirror(obj: Object): Object;
```

Comment:
Yep! See the [handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html).